bucket/vial.json

```
@@ -0,0 +1,21 @@
+{
+  "version": "0.6",
+  "description": "A graphical keyboard configuration tool for QMK powered keyboards.",
+  "homepage": "https://get.vial.today/",
+  "license": "GPL-2",
```

```suggestion
  "license": "GPL-2.0-only",
```
See [SPDX license identifiers](https://spdx.org/licenses/).